szperacz/run.py

```diff
@@ -18,4 +18,23 @@
 
     files = discover_files(path)
 
-    print(files)
+    def found(files):
+        print(f'Images found: {len(files)}\n---')
+        for i in files:
+            print(f'{i}')
+
+        return '---'
+
+
+    def not_found(files):
+        return 'Images found: 0'
+
+
+    def founded(files):
+        if len(files) > 0:
```

Comment:
Please be more `pythonic` and use something like:

```if len(files):
   ...
```

In Python, every value above 0 returns True, capitalize on that.